client/flutter/lib/pages/home_page.dart

```diff
@@ -171,47 +171,83 @@ class _HomePageState extends State<HomePage> {
+                      fontSize: 20,
+                    ),
+                  ),
+                  trailing: Icon(
+                    Icons.arrow_forward_ios,
+                    color: Color(0xFFC9CDD6),
+                  ),
+                  onTap: () {
+                    analytics.logShare(
+                        contentType: 'App',
+                        itemId: null,
+                        method: 'Website link');
+                    Share.share(
+                        S.of(context).commonWhoAppShareIconButtonDescription);
+                  },
                 ),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () {
-                  analytics.logShare(
-                      contentType: 'App', itemId: null, method: 'Website link');
-                  Share.share(
-                      S.of(context).commonWhoAppShareIconButtonDescription);
-                },
               ),
-              Divider(),
-              ListTile(
-                leading: Icon(Icons.settings, color: Color(0xffCA6B35)),
-                title: Text(
-                  S.of(context).homePagePageSliverListSettings,
-                  style: TextStyle(
-                      color: Color(0xffCA6B35),
-                      fontWeight: FontWeight.w600,
-                      fontSize: 20),
+              Divider(height: 1),
+              Material(
+                color: Colors.white,
+                child: ListTile(
+                  contentPadding: EdgeInsets.symmetric(
+                    horizontal: 16,
+                    vertical: 8,
+                  ),
+                  leading: Icon(Icons.settings, color: Color(0xffCA6B35)),
+                  title: Text(
+                    S.of(context).homePagePageSliverListSettings,
+                    style: TextStyle(
+                        color: Color(0xffCA6B35),
+                        fontWeight: FontWeight.w600,
+                        fontSize: 20),
+                  ),
+                  trailing: Icon(
+                    Icons.arrow_forward_ios,
+                    color: Color(0xFFC9CDD6),
+                  ),
+                  onTap: () => Navigator.of(context)
+                      .push(MaterialPageRoute(builder: (c) => SettingsPage())),
                 ),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () => Navigator.of(context)
-                    .push(MaterialPageRoute(builder: (c) => SettingsPage())),
               ),
-              Divider(),
-              ListTile(
-                title: Text(S.of(context).homePagePageSliverListAboutTheApp),
-                trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () {
-                  _logAnalyticsEvent('About');
-                  return Navigator.of(context)
-                      .push(MaterialPageRoute(builder: (c) => AboutPage()));
-                },
+              Divider(height: 1),
+              Material(
```

Comment:
Ah cool that this also works! I didn't want to complain before about `Container` because that's a personal dislike 😛